erpnext_documentation/www/docs/user/manual/en/healthcare/therapy_type.md

```diff
@@ -0,0 +1,77 @@
+<!-- add-breadcrumbs -->
+
+# Therapy Type
+
+As therapists, you might be prescribing a number of different therapies for a patient right from Intensive Upper Limb Training to Functional Mobility and so on. Every therapy might have a set of exercises. ERPNext helps you templatize the therapies using the "Therapy Type" DocType. With this you can create and link therapies to Healthcare Service Units, add standard rates and Item details for billing and also add exercises according to Body Parts.
```

Comment:
As therapists, you might advise a number of different therapies for a patient right from Intensive Upper Limb Training to Functional Mobility.

Reply:
With this, you can create and link therapies to Healthcare Service Units, add standard rates and Item details for billing, and also add exercises according to Body Parts.